app/helpers/account.ts

```diff
@@ -0,0 +1,28 @@
+import { Server } from "stellar-sdk";
+const server: Server = new Server("https://horizon-testnet.stellar.org");
+
+const loadAccount = async (publicKey: string) => {
+  try {
+    const account = await server.loadAccount(publicKey);
+    return account;
+  } catch (error) {
+    throw new Error("Unfunded account");
+  }
+};
+
+const getBalance = async (publicKey: string): Promise<string | undefined> => {
+  let balance: string | undefined = "0";
+
+  try {
+    const account = await loadAccount(publicKey);
+    balance = account.balances.find(
+      (balance) => balance.asset_type === "native"
+    )?.balance;
+  } catch (error) {
+    throw Error("Unfunded account");
+  }
+
+  return balance;
+};
```

Comment:
This doesn't work?
```suggestion
const getBalance = async (publicKey: string): Promise<string | undefined> => {
  try {
    const account = await loadAccount(publicKey);
    return account.balances.find(
      (balance) => balance.asset_type === "native"
    )?.balance;
  } catch (error) {
    throw Error("Unfunded account");
  }
};
```

Reply:
I have changed this function to avoid using unnecessary variables as you have pointed out.